api/api_private_account.go

```diff
@@ -485,31 +485,29 @@ func (s *PrivateAccountAPI) Sign(ctx context.Context, data hexutil.Bytes, addr c
 //
 // https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal_ecRecover
 func (s *PrivateAccountAPI) EcRecover(ctx context.Context, data, sig hexutil.Bytes) (common.Address, error) {
-	pubkey, err := klayEthEcRecover(data, sig)
+	if len(sig) != crypto.SignatureLength {
```

Comment:
How about wrapping L488 to L498 to a function? then L588-596 can be short

Reply:
I made the function first. We need an additional function that cover followings
```go
// Transform yellow paper V from 27/28 to 0/1
sig[crypto.RecoveryIDOffset] -= 27
```
We can name it "transformRecoverIdOffset". But I don't want to add some util property function in this file.
So I just wrote that code twice.